R/resolve_state.R

```diff
@@ -62,7 +62,7 @@ resolve_state.default <- function(x) {
 #' @rdname resolve_state
 #' @keywords internal
 #' @export
-resolve_state.default_filter <- function(x, filter_state) { # nolint
+resolve_state.default_filter <- function(x) { # nolint
```

Comment:
this was an obvious error on the main. `resolve_state` is never called with two arguments

Reply:
looks like our recent efforts of converting some notes to errors pays-off :)